simulators/portal-bridge-tests/src/main.rs

```diff
@@ -0,0 +1,180 @@
+use ethportal_api::HistoryContentKey;
+use ethportal_api::HistoryContentValue;
+use ethportal_api::PossibleHistoryContentValue;
+use ethportal_api::{Discv5ApiClient, HistoryNetworkApiClient};
+use hivesim::{dyn_async, Client, NClientTestSpec, Simulation, Suite, Test, TestSpec};
+use itertools::Itertools;
+use portal_bridge::bridge::Bridge;
+use portal_bridge::execution_api::ExecutionApi;
+use portal_bridge::mode::BridgeMode;
+use portal_bridge::pandaops::PandaOpsMiddleware;
+use serde_json::Value;
+use tokio::time::{sleep, Duration};
+use trin_validation::accumulator::MasterAccumulator;
+use trin_validation::oracle::HeaderOracle;
+
+#[tokio::main]
+async fn main() {
+    tracing_subscriber::fmt::init();
+
+    let mut suite = Suite {
+        name: "portal-bridge-tests".to_string(),
```

Comment:
I think it will be less confusing if we explicitly say that this tests the Trin bridge, because portal may have more than one bridge implementation.

```suggestion
        name: "trin-bridge-tests".to_string(),
```

Reply:
Resolved